Init notebook

stephjuneau opened the pull request:
Here's a simplified version of the notebook that is *not yet final* but I'd like to hear what you think about moving the joint query with LS DR10 to a "BONUS" section at the end below the 3D part, and only keep simple queries to SDSS and DESI for the large scale structure and SPARCL part.

If yall agree then I can also update the Goals, Summary and clean up... If you disagree then there's still time to reverse!!!!

Also if you agree maybe we can merge this one, then David can add the 3D part, then I can make the rest of the changes? (going to ADASSx now to hang the posters)

stephjuneau committed:
Early draft of demo notebook

stephjuneau committed:
notebook with SDSS+DESI LSS

jacquesalice reviewed (approved):
Looks good to me!

stephjuneau committed:
updated title, ToC, summary

stephjuneau commented:
Hi @jacquesalice , @kareninysimba  (Cc @rnikutta ), I just updated the Title, Table of content, Goals, Summary and some of the text in the sections. I haven't touched the 3D part.

What is still needed:
1- adding the final image screenshot (me)
2- updating the 3D part with the DESI region (David)
3- adding References / notebooks / data acknowledgments at the end (can someone help? Alice or David?)

kareninysimba reviewed (changes requested):
Everything runs fine!
A change needed is the coords. of the 3D data query to the one you suggested and I showed in our run-through.
Can you please change the last line of the query after "Data query and preparation" from
(160,45,10)
to
(220,40.,10)
so we keep things simpler and not that many branches and stuff.
Thanks!

stephjuneau committed:
updated notebook and rendedered copy

stephjuneau committed:
fixed comment on GIF

stephjuneau commented:
OK, thanks @kareninysimba , I made the changes that you sent me and also added a RENDERED copy as a backup but cleared the main notebook from all output. It all worked including the 3D part when I tried it but I removed the rendered 3D plot to save some memory. Does it look good enough to merge?

kareninysimba reviewed (approved):
It seems to run fine!
